nixos/tests/postgis.nix

```diff
@@ -26,5 +26,9 @@ import ./make-test-python.nix ({ pkgs, ...} : {
     master.succeed("sudo -u postgres psql -c 'CREATE EXTENSION postgis;'")
     master.succeed("sudo -u postgres psql -c 'CREATE EXTENSION postgis_raster;'")
     master.succeed("sudo -u postgres psql -c 'CREATE EXTENSION postgis_topology;'")
+    master.succeed("sudo -u postgres psql -c 'select postgis_version();'")
+    master.succeed("[ \"$(sudo -u postgres psql --no-psqlrc --tuples-only -c 'select postgis_version();')\" = \" 3.5 USE_GEOS=1 USE_PROJ=1 USE_STATS=1\" ]")
```

Comment:
Please avoid hardcoding the version (`3.5`)

Reply:
Changed by using the package version directly.